adsingestp/parsers/adsfeedback.py

```diff
@@ -0,0 +1,83 @@
+import json
+import logging
+
+from adsingestp import utils
+from adsingestp.parsers.base import BaseBeautifulSoupParser
+
+logger = logging.getLogger(__name__)
+
+
+class ADSFeedbackParser(BaseBeautifulSoupParser):
+    def __init__(self, json_string):
```

Comment:
I think we have moved toward not requiring that data be loaded in __init__.  Some of the parsers have that option, but I think our standard is to load when the .parse() method is called.  So two things here: one, give the json_string a default value of None so that you don't have to include it, and only initialize it if it's not None; two, add an argument to .parse() where you can pass json_string.